tests/blacken_docs_test.py

```diff
@@ -153,6 +153,102 @@ def test_format_src_rst():
     )
 
 
+def test_format_src_rst_sphinx_doctest():
+    before = (
+        '.. testsetup:: group1\n'
+        '\n'
+        '    mock = SomeMock( )\n'
+        '\n'
+        '.. testcleanup:: group1\n'
+        '\n'
+        '    mock.stop( )\n'
+        '\n'
+        '.. doctest:: group1\n'
+        '\n'
+        '   >>> parrot.voom( 3000 )\n'
+        '   This parrot wouldn\'t voom if you put 3000 volts through it!\n'
+        '\n'
+    )
+    after, _ = blacken_docs.format_str(before, BLACK_MODE)
+    assert after == (
+        '.. testsetup:: group1\n'
+        '\n'
+        '    mock = SomeMock()\n'
+        '\n'
+        '.. testcleanup:: group1\n'
+        '\n'
+        '    mock.stop()\n'
+        '\n'
+        '.. doctest:: group1\n'
+        '\n'
+        '   >>> parrot.voom(3000)\n'
+        '   This parrot wouldn\'t voom if you put 3000 volts through it!\n'
+        '\n'
+    )
+
+
+def test_format_src_rst_sphinx_doctest_testcode():
+    before = (
+        'The parrot module\n'
+        '=================\n'
+        '\n'
+        '.. testsetup:: *\n'
+        '\n'
+        '   import parrot  \n'
+        '\n'
+        'The parrot module is a module about parrots.\n'
+        '\n'
+        'Doctest example:\n'
+        '\n'
+        '.. doctest::\n'
+        '\n'
+        '   >>> parrot.voom( 3000 )\n'
+        '   This parrot wouldn\'t voom if you put 3000 volts through it!\n'
+        '\n'
+        'Test-Output example:\n'
+        '\n'
+        '.. testcode::\n'
+        '\n'
+        '   parrot.voom( 3000 )\n'
+        '\n'
+        'This would output:\n'
+        '\n'
+        '.. testoutput::\n'
+        '\n'
+        '   This parrot wouldn\'t voom if you put 3000 volts through it!\n'
```

Comment:
this test can be much shorter, no need to have all the other fluff here -- just get straight to the point and demonstrate what you're meaning to demonstrate -- this means you can get rid of the `testoutput` and the intermediate text, etc.

Reply:
I figured it made sense to use the exact, entire example from the sphinx doctest documentation.  Instead, I'll just remove this one and add the the `testcode` to the previous one so it's all in 1 test.